getting_started.md

```diff
@@ -0,0 +1,253 @@
+config.video[1].camera.settings.shape = (1280, 720)
+```
+
+Now we'll configure each output, or sink device.
+For both simulaed cameras, we'll be writing to Zarr, a format which supports chunked arrays.
+
+
+```python
+config.video[0].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Zarr")
+
+# what file or directory to write the data to
+config.video[0].storage.settings.filename = "output1.zarr"
+
+# where applicable, how large should a chunk file get before opening the next chunk file
+config.video[0].storage.settings.bytes_per_chunk = 32 * 2**20  # 32 MiB chunk sizes
+```
+
+
+```python
+config.video[1].storage.identifier = dm.select(acquire.DeviceKind.Storage, "Zarr")
+
+# what file or directory to write the data to
+config.video[1].storage.settings.filename = "output2.zarr"
+
+# where applicable, how large should a chunk file get before opening the next chunk file
+config.video[1].storage.settings.bytes_per_chunk = 64 * 2**20  # 64 MiB chunk sizes
+```
+
+Finally, let's specify how many frames to generate for each camera before stopping our simulated acquisition.
+We also need to register our configuration with the runtime.
+
+If you want to let the runtime just keep acquiring forever, you can set `max_frame_count` to 0.
+
+```python
+config.video[0].max_frame_count = 100
+config.video[1].max_frame_count = 150
+
+config = runtime.set_configuration(config)
+```
+
+---
+**NOTE**
+
+If you are using a Jupyter notebook for this tutorial, clear output from previous runs with:
+
+```python
+import os
+import shutil
+
+if config.video[0].storage.settings.filename in os.listdir("."):
+    shutil.rmtree(config.video[0].storage.settings.filename)
+    
+if config.video[1].storage.settings.filename in os.listdir("."):
+    shutil.rmtree(config.video[1].storage.settings.filename)
+```
+---
+## Acquire Data
+
+To start aquiring data:
+
+```python
+runtime.start()
+```
+
+Acquisition happens in a separate thread, so at any point we can check on the status by calling `runtime.get_state()`.
+
+
+```python
+runtime.get_state()
+```
+
+Finally, once we're done acquiring, we call `runtime.stop()`.
+This method will wait until you've reached the maximum number of frames specified in `config.video[0].max_frame_count`.
+
+```python
+runtime.stop()
+```
+
+## Visualizing the Data with napari
+
+Let's take a look at what we've written.
+We'll load each Zarr dataset as a Dask array and inspect its dimensions, then we'll use napari to view it.
+
+```python
+import dask.array as da
+import napari
+```
+
+```python
+data1 = da.from_zarr(config.video[0].storage.settings.filename, component="0")
+data1
+```
+
+```python
+data2 = da.from_zarr(config.video[1].storage.settings.filename, component="0")
+data2
+```
+
+```python
+viewer1 = napari.view_image(data1)
+```
+
+```python
+viewer2 = napari.view_image(data2)
+```
+
+## Conclusion
+
+For more examples of using Acquire, check out our [tutorials page](). #ADD LINK
```

Comment:
> #ADD LINK

👀

Reply:
I'll add link once we have the tutorials page setup.